query/src/test/resources/nrt-performance-writer-infinispandirectory.xml

```diff
@@ -31,12 +31,12 @@
             <!-- Write indexes in Infinispan -->
             <property name="directory.type">local-heap</property>
             <!-- Enable sharding on writers -->
-            <property name="index_defaults.sharding.strategy">hash</property>
-            <property name="index_defaults.sharding.number_of_shards">6</property>
+            <property name="sharding.strategy">hash</property>
+            <property name="sharding.number_of_shards">6</property>
```

Comment:
FYI, Hibernate Search the routing keys for sharding when they are provided, not the document IDs.
Infinispan uses the segment IDs as routing keys.
Not sure it changes anything, but I thought you would want to know.

Reply:
Regarding shards: the plan is to not expose it to the user whatsoever.